src/include/migraphx/matcher.hpp

```diff
@@ -428,6 +428,62 @@ void find_matches(Mod& mod, Ms&&... ms)
     }
 }
 
+/// Find matches for an instruction in the module for per section of matchers
+template <class Mod, class... Ms>
+void find_matches(int trace_mod, Mod& mod, instruction_ref ins, Ms&&... ms)
```

Comment:
You can keep just one overload and make line 427 do `find_matches(0, mod, ins, ms...);`

Reply:
Done